### ISSUE TITLE
Error installing latest version with npm

### ISSUE THREAD

**Issue by HalfdanJ** [agershun/alasql#45]
Trying to install 0.0.36 alasql with nom on mac os 10.9 returns the following error:

````
npm ERR! Error: ENOENT, chmod '/Users/drexjojo/Development/QCLogServer/node_modules/alasql/bin/alacon.js'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Darwin 13.3.0
npm ERR! command "node" "/usr/local/bin/npm" "install"
npm ERR! cwd /Users/drexjojo/Development/QCLogServer
npm ERR! node -v v0.10.35
npm ERR! npm -v 1.4.28
npm ERR! path /Users/drexjojo/Development/QCLogServer/node_modules/alasql/bin/alacon.js
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! not ok code 0
````

Installing 0.0.35 works without any problems

**Comment by agershun** [comment]
Thank you very much for the issue! I can fix it tonight. 

As a temporary solution: download Alasql as zip file and unpack it to node_modules/alasql directory of your project.

**Comment by agershun** [comment]
The problem is fixed in version 0.0.37. Thank you!
